Makefile

```diff
@@ -60,7 +60,7 @@ test-e2e:
 
 .PHONY: test-unit
 test-unit:
-	go test -v ./pkg/...
+	go test -v -coverprofile=coverage.txt -covermode=atomic ./pkg/...
```

Comment:
Why `-covermode atomic`? The `go help testflag` manual seems to imply that by default some reasonable values are used for this flag; is it an explicit requirement of codecov.io? If yes, please add a comment line above l.63 with an explanation.

Reply:
I followed the Go example for codecover.io. This is where I got `covermode atomic` from.

https://github.com/codecov/example-go#codecov-go-example

It looks like they recommend `atomic` or `count`. Let me now which you prefer.

Reply:
@akavel if you have some time to let me know what you think about this. That would be great. I'd ideally like to merge this tomorrow.